snappiershot/snapshot/_file.py

```diff
@@ -187,8 +187,9 @@ def _find_snapshots(file_contents: Dict, metadata: SnapshotMetadata) -> List[Dic
                 return function_snapshot[SnapshotKeys.snapshots]
 
         # If no sub-section exists, create it.
+        encoded_metadata = cast(dict, default_encode_value(metadata.as_dict()))
```

Comment:
this is just for type-hinting purposes right?

Reply:
yes